doc/Command-Reference.md

```diff
@@ -3559,6 +3587,67 @@ This command displays the details of Rx & Tx priority-flow-control (pfc) for all
   root@sonic:~# sonic-clear pfccounters
   ```
 
+**show pfc asymmetric**
+
+This command displays the status of asymmetric PFC for all interfaces or a given interface.
+
+- Usage:
+  ```
+  show pfc asymmetric
+  show pfc asymmetric <interface>
```

Comment:
Combine these into one line, indicating that the <interface> parameter is optional, like this:

```
  show pfc asymmetric [<interface>]
```

Reply:
@jleveque Fixed